packages/hint-compat-api/src/compat-api-css-next.ts

```diff
@@ -0,0 +1,149 @@
+/**
+ * @fileoverview Hint to validate if the CSS features of the project are not broadly supported
+ */
+
+import { Category } from 'hint/dist/src/lib/enums/category';
+import { HintScope } from 'hint/dist/src/lib/enums/hintscope';
+import { HintContext } from 'hint/dist/src/lib/hint-context';
+import { IHint, HintMetadata, ProblemLocation } from 'hint/dist/src/lib/types';
+import { StyleParse } from '@hint/parser-css/dist/src/types';
+import { CompatApi, userBrowsers, CompatCSS } from './helpers';
+import { BrowserSupportCollection } from './types';
+import { SimpleSupportStatement, SupportStatement } from './types-mdn.temp';
+import { browserVersions } from './helpers/normalize-version';
+
+/*
+ * ------------------------------------------------------------------------------
+ * Public
+ * ------------------------------------------------------------------------------
+ */
+
+type UserPrefixes = {
+    [key: string]: boolean;
+};
+
+export default class implements IHint {
+
+    public static readonly meta: HintMetadata = {
+        docs: {
+            category: Category.interoperability,
+            description: `Hint to validate if the CSS features of the project are not broadly supported`
+        },
+        id: 'compat-api-css-next',
+        schema: [],
+        scope: HintScope.any
+    }
+
+    private mdnBrowsersCollection: BrowserSupportCollection;
+    private compatApi: CompatApi;
+    private compatCSS: CompatCSS;
+    private userPrefixes: UserPrefixes = {};
+
+    public constructor(context: HintContext) {
+        const isCheckingNotBroadlySupported = true;
+
+        this.mdnBrowsersCollection = userBrowsers.convert(context.targetedBrowsers);
+        this.compatApi = new CompatApi('css', this.mdnBrowsersCollection, isCheckingNotBroadlySupported);
+        this.compatCSS = new CompatCSS(context, (...params) => {
```

Comment:
This is  not going to work :(

Reply:
why?

Reply:
ok after talk we are going to change arrow functions to bind(this), because the ideal case is one line arrow functions but are not allowed by linting. @borgitas21 we can talk about this tomorrow.

We can talk in the future with @molant and @sarvaje to allow one line lambdas, but not important at this moment :)

How we understand what is better:
ok
```
this.compatCSS = new CompatCSS(context, (...params) => {
    this.testFeatureIsSupportedInBrowser(...params);
});
```

better
```
new CompatCSS(context, this.testFeatureIsSupportedInBrowser.bind(this));
```

best
```
new CompatCSS(context, (...x) => this.testFeatureIsSupportedInBrowser(...x));
```

Reply:
IMO, we should go with bind even if we enable one line lambdas.

The effect is the same, we are creating a new function in both cases, but in the case of the `bind`, you don't care about the parameters or anything else.

At least for me, `bind` is cleaner than one line lambda.

@webhintio/contributors thoughts?

Reply:
Implemented as part of #14